packages/shared/ReactTypes.js

```diff
@@ -79,9 +79,12 @@ export type ReactContext<T> = {
   $$typeof: Symbol | number,
   Consumer: ReactContext<T>,
   Provider: ReactProviderType<T>,
+
   unstable_read: () => T,
+  unstable_set: (value: T, callback: (() => mixed) | void | null) => void,
```

Comment:
`writeDefault` (or maybe `writeGlobal` / `writeBase`)

`Context.write()` implies to me like it would write to the closest provider (just like, if i'm not terribly mistaken, `Context.read` reads from the closest provider).

Reply:
I agree `read` / `write` mismatch is confusing.

Reply:
`read` is not going to be a public API anyway so we can rename it to something more random.

`pushDefault` or `writeDefault` could work too.